src/components/dictionaryConcepts/components/ActionButtons.jsx

```diff
@@ -23,6 +23,14 @@ const ActionButtons = ({
     <React.Fragment>
       {source !== CUSTOM_SOURCE && (
         <React.Fragment>
+          <a
+            href={`${TRADITIONAL_OCL_HOST}${url}`}
+            target="_blank"
```

Comment:
Is this a formatting change?

Reply:
Technically, no, but it is a field swap so that we have the remove and retire buttons are in the same column.
New ticket?